test/test_model.py

```diff
@@ -103,7 +107,8 @@ def check_waveform(assertion, generated_waveform):
 
     # Expect most of the power to be at the 3 frequencies we trained
     # on.
-    assertion(expected_power, 0.9 * power_sum)
+    assertion(expected_power, 0.75 * power_sum)
```

Comment:
I think the failure to pass test may be symptom of a deeper problem. Please see comment on receptive field size.